packs/data/pathfinder-bestiary-3.db/animated-colossus.json

```diff
@@ -29,8 +29,11 @@
             "allSaves": {
                 "value": ""
             },
+            "hardness": {
+                "value": 15
+            },
             "hp": {
-                "details": "Hardness 15",
+                "details": "",
```

Comment:
?

Reply:
Well, NPCs do have their own hardness field now, so I entered the value there:
![image](https://user-images.githubusercontent.com/66370966/175209657-cfcf4df0-1476-4e18-affd-9d7bb9dd872d.png)

Reply:
Ah, yes.